app/services/form_validator.rb

```diff
@@ -1,5 +1,7 @@
 class FormValidator
   attr_reader :errors
+  MAX_TEXT_LENGTH = 250
```

Comment:
Are these setting the max input value that a user can enter or the max length for the name (I'm assuming the former). Can we use an HTML5 attribute on the fields to make sure that there's browser support for those limits, too? (I looked but didn't see them)